src/Shared/UnitTests/FileUtilities_Tests.cs

```diff
@@ -283,7 +287,14 @@ public void HasExtension_UsesOrdinalIgnoreCase()
 
                 var result = FileUtilities.HasExtension("foo.ini", new string[] { ".INI" });
 
-                Assert.True(result);
+                if (FileUtilities.GetIsFileSystemCaseSensitive())
+                {
+                    Assert.False(result);
```

Comment:
```suggestion
                    result.ShouldBeFalse();
```
Same below. Alternatively, you could condense these to:
(result ^ FileUtilities.GetIsFileSystemCaseSensitive()).ShouldBeTrue();